### ISSUE TITLE
Result bug when using the count aggregation?

### ISSUE THREAD

**Issue by dmichel1** [spotify/heroic#256]
Hello,

Not sure if this is a bug or expected behavior but when sending the below query to Heroic that uses a count aggregation, it will always produce two results. If I change the aggregation to `average`, I get a single result.

Note, the sampling value is the difference between (end-start), and that is why I expect to receive only a single result back.

*Request*
```
{
  "filter": [
    "and",
    [
      "key",
      "data-counters-publishing"
    ]
  ],
  "aggregators": [
    {
      "type": "group",
      "each": [
        {
          "type": "count",
          "sampling": {
            "unit": "SECONDS",
            "value": 604800
          }
        }
      ],
      "of": null
    }
  ],
  "range": {
    "type": "absolute",
    "start": 1490122567000,
    "end": 1490727367000
  },
  "features": [
    "com.spotify.heroic.distributed_aggregations"
  ]
}
```

*Response*
```
...
"values": [
  [
  1489622400000,
  0
],,
  [
  1490227200000,
  10080
],
],
"key": "data-counters-publishing",
```

**Comment by udoprog** [comment]
Related to https://github.com/spotify/heroic/pull/230 - which is motivated by [rfc#2](https://github.com/spotify/heroic/blob/master/docs/rfcs/0002-aligned-timestamp.md).

This particular issue should have been addressed in #230.

**Comment by udoprog** [comment]
Feel free to close if you agree.

**Comment by dmichel1** [comment]
@udoprog awesome, it does indeed look to be addressed in #230. has this been rolled out internally?

Thanks
